assets/css/style.css

```diff
@@ -91,4 +108,11 @@ h2,
 
 #error-header {
     background-color: #E82F1C;
-}
\ No newline at end of file
+}
+
+
+/* @media all and (max-width: 768px) {
```

Comment:
Can pry remove this since its commented out.

Reply:
fixed